experience.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Daria's Experience</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+  </head>
+
+    <body>
+      <img class="main" src="images/daria_logo.svg.png">
```

Comment:
The name of the image file name has mixed cases, while you are defining a file name that is all lowercase. Thats' why the image is not loading. In the future try to name and your files in lowercase. 🔨